lsp/src/main/scala/org/ensime/lsp/rpc/RpcFormats.scala

```diff
@@ -133,8 +141,8 @@ object RpcFormats {
 
   implicit val JsonRpcRequestMessageBatchFormat
     : RootJsonFormat[JsonRpcRequestMessageBatch] =
-    RpcConversions.JsonRpcRequestMessageBatchFormat
+    RpcConversions.request
   implicit val JsonRpcResponseMessageBatchFormat
     : RootJsonFormat[JsonRpcResponseMessageBatch] =
-    RpcConversions.JsonRpcResponseMessageBatchFormat
+    RpcConversions.response
```

Comment:
I think it's okay to have such long names for `*BatchFormat`, there're quite rare as I understand